app.py

```diff
@@ -35,30 +35,10 @@
 def get():
     lists = dashboard.get()
     sessions = {'first','second'}
-    return render_template('dashboard.html', lists = lists, sessions=sessions)  
-
-@app.route('/removeDashboard/')
-@app.route('/removeDashboard/', methods=['GET','POST'])
-def get_rm():
-    lists = removeDashboard.get_rm()
-    return render_template('removeDashboard.html', lists = lists)
-
-app.add_url_rule('/addStudent/',
-                view_func=AddStudent.as_view('addStudent'),
-                methods=['GET', 'POST'])
-                
-app.add_url_rule('/addTeam/',
-                view_func=AddTeam.as_view('addTeam'),
-                methods=['GET', 'POST'])
-
-app.add_url_rule('/removeStudent/',
-                view_func=RemoveStudent.as_view('removeStudent'),
-                methods=['GET', 'POST'])
-
-app.add_url_rule('/removeTeam/',
-                view_func=RemoveTeam.as_view('removeTeam'),
-                methods=['GET', 'POST'])
+    form = dashboard.Form()
+    team_names = dashboard.get_teams()
+    member_names = dashboard.get_members()
+    return render_template('dashboard.html', lists=lists, sessions=sessions, team_names=team_names, member_names=member_names, form=form) 
```

Comment:
Can we put all of this into a MethodView class (like the ones in removeClass.py) and use an app.add_url_rule(...) command in app.py to add it to the app?